io.github.ja2-stracciatella.appdata.xml

```diff
@@ -0,0 +1,57 @@
+    <id>io.github.ja2-stracciatella</id>
+    <launchable type="desktop-id">io.github.ja2-stracciatella.desktop</launchable>
+    <name>Jagged Alliance 2 Stracciatella</name>
+    <summary>An improved, cross-platform, stable Jagged Alliance 2 runtime</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/SFI%20Source%20Code%20license%20agreement.txt</project_license>
+    <url type="homepage">https://ja2-stracciatella.github.io</url>
+    <url type="help">https://ja2-stracciatella.github.io/how-to-run/</url>
+    <url type="vcs-browser">https://github.com/ja2-stracciatella/ja2-stracciatella/</url>
+    <description>
+      <p>
+        Jagged Alliance 2 is a tactical turn-based role-playing game. The game
+        takes place in the fictional country of Arulco, which has been ruled by
+        the ruthless monarch Deidranna for several years. The player is put in
+        control of hired mercenaries and with aid of local citizens and militia
+        must reclaim Arulco's cities and ultimately defeat Deidranna.
+      </p>
+      <p>
+        The Jagged Alliance 2 Stracciatella runtime makes Jagged Alliance 2
+        available on a wide range of platforms, improves its stability,
+        fixes bugs and provides a stable platform for mod development.
+      </p>
+      <p>
+        Data files from the original game are still required and will be used by
+        JA2-Stracciatella.
+      </p>
+    </description>
+    <categories>
+      <category>Game</category>
+      <category>RolePlaying</category>
+      <category>ActionGame</category>
+      <category>StrategyGame</category>
+      <category>Adult</category>
+    </categories>
+    <update_contact>sebastian@swsnr.de</update_contact>
+    <releases>
+        <release version="0.20.0" date="2022-11-14">
+          <url>https://ja2-stracciatella.github.io/2022/11/14/release-0.20.0.html</url>
+        </release>
+    </releases>
+    <!-- Generated with https://hughsie.github.io/oars/generate.html -->
+    <content_rating type="oars-1.1">
+      <content_attribute id="violence-fantasy">intense</content_attribute>
+      <content_attribute id="violence-bloodshed">intense</content_attribute>
+      <content_attribute id="violence-sexual">intense</content_attribute>
+      <content_attribute id="violence-desecration">intense</content_attribute>
+      <content_attribute id="drugs-alcohol">moderate</content_attribute>
+      <content_attribute id="drugs-narcotics">moderate</content_attribute>
+      <content_attribute id="drugs-tobacco">mild</content_attribute>
+      <content_attribute id="sex-nudity">mild</content_attribute>
+      <content_attribute id="sex-themes">moderate</content_attribute>
+      <content_attribute id="language-profanity">intense</content_attribute>
+      <content_attribute id="language-humor">intense</content_attribute>
+    </content_rating>
```

Comment:
this is missing required screenshots.

Reply:
Upstream doesn't have screenshots in their repo or on their Github page.  Where should I get screenshots from?  Should I make one and add it to this repo?

Reply:
I found an (unlinked) screenshot on the website; should be good now.